tests/specs/functional.spec.js

```diff
@@ -0,0 +1,39 @@
+import {test, expect} from '@playwright/test';
+import {Main} from '../pageObjects/Main.js';
+import fixture from '../fixtures/main.js';
+
+
+test.describe('functional test', () => {
+    test('check visibility of elements', async ({page, baseURL}) => {
+        await page.goto(baseURL);
+
+        const $titles = await page.locator(fixture.parseMessages.title.selector).allInnerTexts();
```

Comment:
What's the point to use universal selector in fixtures? 
It looks bad when a test case uses the selector from `fixture.parseMessages.title` to check the text equality of `fixture.createMessages.title` and `fixture.logs.title`. Also, the selectors from `fixture.createMessages.title` and `fixture.logs.title` are never used.

Just use `await page.locator('h5').allInnerTexts();` or check items one by one

Reply:
ok